pkg/api/errors.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import "fmt"
+
+// TaskMissingError implements an exaplainable error.
+type TaskMissingError struct {
+	app  string
+	slug string
+}
+
+// Error implementation.
+func (err TaskMissingError) Error() string {
+	return fmt.Sprintf("task with slug %q does not exist", err.slug)
```

Comment:
Should we capitalize error messages? Sort of tricky if we don't know if it'll get prefixed by something else I suppose...

```suggestion
	return fmt.Sprintf("Task with slug %q does not exist", err.slug)
```

Reply:
Yeah, exactly why they're not supposed to be capitalized in go, this is why you see errors like:

```
net: <msg> net/http: <msg>
```